website/docs/rules/flutter/avoid-use-expanded-as-spacer.md

```diff
@@ -0,0 +1,39 @@
+# Avoid use Expanded widget instead of Spacer
+
+## Rule id {#rule-id}
+
+avoid-use-expanded-as-spacer
+
+## Severity {#severity}
+
+Style
+
+## Description {#description}
+
+The rule should detect Expanded widgets that contain empty SizedBox/Container and propose to replace them with Spacer()
```

Comment:
The rule detects Expanded widgets that contain empty SizedBox/Container and proposes to replace them with the Spacer widget.